packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -168,36 +193,84 @@ function SideNavGroup({
   children,
   collapsible = false,
   initialCollapsed = true,
+  glyph,
+  className,
   ...rest
 }: SideNavGroupProps) {
   const [open, setOpen] = React.useState(!initialCollapsed);
   const nodeRef = React.useRef(null);
   const ulRef = React.useRef<HTMLUListElement>(null);
   const { usingKeyboard: showFocus } = useUsingKeyboardContext();
+  const menuGroupLabelId = useMemo(
+    () => sideNavGroupIdAllocator.generate(),
+    [],
+  );
+  const menuId = useMemo(() => sideNavGroupIdAllocator.generate(), []);
+
+  const isActiveGroup: boolean = useMemo(() => {
+    return React.Children.toArray(children).some(child => {
+      return isComponentType(child, 'SideNavItem') && child.props.active;
+    });
+  }, [children]);
+
+  const accessibleGlyph =
+    glyph && (isComponentGlyph(glyph) || isComponentType(glyph, 'Icon'))
+      ? React.cloneElement(glyph, {
```

Comment:
does this need to be a ternary?

Reply:
Not technically, but I did prefer this approach largely because of type inference. We don't need to type the variable explicitly since it's part of the same statement.

If you feel strongly, I can update this.